ctpolicy/ctpolicy.go

```diff
@@ -291,16 +153,21 @@ func (ctp *CTPolicy) getOperatorSCTs(ctx context.Context, cert core.CertDER, exp
 		// thereby causing all of our getOne sub-goroutines to be cancelled.
 		return nil, berrors.MissingSCTsError("failed to get 2 SCTs before ctx finished: %s", ctx.Err())
 	}
-	return nil, berrors.MissingSCTsError("failed to get 2 SCTs, got error(s): %s", strings.Join(errs, "; "))
+	return nil, berrors.MissingSCTsError("failed to get 2 SCTs, got %d error(s): %s", len(errs), strings.Join(errs, "; "))
 }
 
 // submitAllBestEffort submits the given certificate or precertificate to every
 // log ("informational" for precerts, "final" for certs) configured in the policy.
 // It neither waits for these submission to complete, nor tracks their success.
-func (ctp *CTPolicy) submitAllBestEffort(blob core.CertDER, precert bool, expiry time.Time) {
+func (ctp *CTPolicy) submitAllBestEffort(blob core.CertDER, isPrecert bool, expiry time.Time) {
 	logs := ctp.finalLogs
-	if precert {
+	result := succeeded
+	logType := cert
+	logKind := sct
```

Comment:
All calls to `submitAllBestEffort` are informational -- either we're submitting a precert but not using the SCTs, or we're submitting a final cert. Either way, we don't care about getting SCTs back.
```suggestion
	logKind := inform
```